openml/study/functions.py

```diff
@@ -27,9 +27,11 @@ def get_study(study_id):
     creator = result_dict['oml:creator']
     tags = []
     for tag in _multitag_to_list(result_dict, 'oml:tag'):
-        tags.append({'name': tag['oml:name'],
-                     'window_start': tag['oml:window_start'],
-                     'write_access': tag['oml:write_access']})
+        current_tag = {'name': tag['oml:name'],
+                       'write_access': tag['oml:write_access']}
```

Comment:
the tags that are associated with a study determine which entities belong to the study. window start and write_access are both specifying this in more detail